ictv/plugins/editor/templates/editor.html

```diff
@@ -475,31 +488,32 @@ <h4 class="modal-title">Your slide looks a bit overweight.</h4>
                         }
                     }
                     if(!respectsAllLimits) {
-                        $$('#limit-modal').modal('show');
+                        $('#limit-modal').modal('show');
                     }
                     return respectsAllLimits
                 });
-                $$form.on('mouseenter mouseleave', 'input, .form-group > .cke', function(e) {
+                $form.on('mouseenter mouseleave', 'input, .form-group > .cke', function(e) {
                     var id = this.getAttribute('id');
                     if(id.startsWith('cke_')) {
                         id = id.replace('cke_', '');
                     }
-                    var $$element = $$slide.find('#' + id);
-                    $$element.css({'transition' : 'box-shadow 1s ease'});
+                    var $element = $slide.find('#' + id);
+                    $element.css({'transition' : 'box-shadow 1s ease'});
                     if(e.type == 'mouseenter') {
-                        $$element.addClass('hover');
+                        $element.addClass('hover');
                         setTimeout(function() {
                             render();
                             render();
                         }, 0);
                     } else {
-                        $$element.removeClass('hover');
+                        $element.removeClass('hover');
                     }
                 });
                 render();
                 render();
             } else {
-                window.location = window.location;
+                console.log("wtf");
```

Comment:
:eyes:

Reply:
Woops, this is solved now. 
With my apologies.